packages/application-shell/src/components/back-to-project/back-to-project.spec.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Wanted to have a spec so it can't be changed without knowing/being informed of it.